metron-platform/metron-solr/src/main/java/org/apache/metron/solr/dao/SolrSearchDao.java

```diff
@@ -176,19 +179,23 @@ private SolrQuery buildSearchRequest(
       facetFields.get().forEach(query::addFacetField);
     }
 
-    String collections = searchRequest.getIndices().stream().collect(Collectors.joining(","));
-    query.set("collection", collections);
+    query.set("collection", getCollections(searchRequest.getIndices()));
```

Comment:
This is the key fix for the bug that you found.  When I comment this out and run the tests, the integration test does fail, which is great.  

Unfortunately, I would have expected the `SolrSearchDaoTest` to fail too, but it did not.  If you think it is easy enough to fail a unit test on this condition, it would be a good add.

Reply:
Done